src/pages/index.tsx

```diff
@@ -10,9 +16,113 @@ const HomePage = () => {
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
-      <main>HomePage</main>
+      <main>
+        <PageContainer>
+          <div>
+            <input />
+            <Tags>
+              <Button buttonType="tag">#힐링</Button>
+              <Button buttonType="tag">#이쁜카페</Button>
+              <Button buttonType="tag">#드라이브</Button>
+              <Button buttonType="tag">#맛집</Button>
+              <Button buttonType="tag">#가족여행</Button>
+              <Button buttonType="tag">#혼자여행</Button>
+            </Tags>
+          </div>
+        </PageContainer>
+        <MainContent>
+          <PageContainer>
+            <Title size="md" fontWeight={700} style={{ marginBottom: 30 }}>
+              인기 여행코스
+            </Title>
+            <ul>
+              <CourseItem>
+                <Thumbnail>
+                  <Text size="xs">제주 5코스</Text>
+                  <Title level={3} size={18} ellipsis>
+                    [1박 2일] 제주도 여행 추천! 힐링하고 싶은 사람 모두 모여라!
+                  </Title>
+                </Thumbnail>
+                <CourseInfo>
+                  <Text block ellipsis>
+                    인천공항→ 도렐제주본점 → 서귀포 1번길 → 기타등등의 여행지
```

Comment:
일관성을 위해 한 칸 띄워주시면 좋아보여요 😀😀
```suggestion
                    인천공항 → 도렐제주본점 → 서귀포 1번길 → 기타등등의 여행지
```